src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import "./Introduce.scss";
+
+function Introduce() {
+  function handleInput() {
+    console.log("input 태그 동작");
+  }
+  const navigate = useNavigate();
+
+  return (
+    <div className="Introduce">
```

Comment:
```suggestion
    <div className="introduce">
```
className은 소문자로 시작해야 하는 것이 컨벤션입니다!